.github/workflows/urls-smoke-test.yml

```diff
@@ -39,10 +39,10 @@ jobs:
               uses: actions/github-script@v7
               with:
                   script: |
-                      const { buildPreviewURLComment } = await import('${{ github.workspace }}/tasks/build-preview-urls-comment.js');
+                      const { buildPreviewURLComment } = await import('${{ github.workspace }}/.github/scripts/build-preview-urls-comment.js');
```

Comment:
could we string deliminate the const so we don't have to await the same import twice?